src/client.ts

```diff
@@ -93,6 +98,23 @@ export class Client {
       this.unpackAndApplyStateData(stateData);
     }
   }
+  
+  /**
+   * Get the public key associated with the client's static keypair.
+   * The public key is used for identifying the client to the Lattice.
+   * @internal
+   * @returns Buffer
+   */
+  public get publicKey () {
+    return getPubKeyBytes(this.key);
+  }
+
+  /**
+   * Get the pairing name for this client instance
+   */
+  public get pairingName () {
```

Comment:
I had to do this because `validateConnectedClient` did not have access to `client.name` and also I thought `pairingName` was a more appropriate public name for the thing in question